### PR TITLE
Moving from `App` to `Account` terminology - Part 3

### PR DESCRIPTION
**Pull request by YaronWittenstein**
This is PR number `3` for implementing issue #264

### CODE REVIEW

File: crates/runtime/src/context.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@ pub struct Context {
 
     template_addr: TemplateAddr,
 
-    app_addr: AppAddr,
+    app_addr: AccountAddr,
```
Comment (neysofu) [inline crates/runtime/src/context.rs:22]: Fix this in next PR?

File: crates/runtime/src/context.rs [hunk 2]
```diff
@@ -52,7 +52,7 @@ impl Context {
         gas_limit: Gas,
         storage: AppStorage,
         template_addr: &TemplateAddr,
-        app_addr: &AppAddr,
+        app_addr: &AccountAddr,
```
Comment (neysofu) [inline crates/runtime/src/context.rs:55]: As above

**Review by neysofu — APPROVED** [review]

File: crates/runtime/src/context.rs (hunk 1 above)
Comment (YaronWittenstein) [reply crates/runtime/src/context.rs:22]: yes

File: crates/runtime/src/context.rs (hunk 2 above)
Comment (YaronWittenstein) [reply crates/runtime/src/context.rs:55]: yes

**Pull request merged**